sysl2/sysl/importer/tests-xsd/namespace-simple.sysl

```diff
@@ -12,7 +12,7 @@ testapp [package="package_foo"]:
     # definitions
 
     !type PurchaseOrderType [~xml_root]:
-        ShipTo <: sequence of USAddress(0..2):
+        ShipTo(0..2) <: sequence of USAddress:
```

Comment:
What's driving this change? Constraints apply to types, not to fields.

Reply:
The original wasn't valid according to the parser